### PR TITLE
Minor edits to transmission text

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is just some straight-forward clean-up of punctuation, grammar, etc. 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/08.transmission.md [hunk 1]
```diff
@@ -21,46 +21,47 @@ Contact, droplet, and aerosol transmission are therefore all worth evaluating wh
...
+The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV [@doi:10.1056/NEJMc2004973], and fomite transmission could also play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).
 However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458], and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].
-These type of transmission are likely to contribute to varying extents to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.
+These type of transmission are likely to contribute, to varying extents, to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.
```
Comment (cgreene) [inline content/08.transmission.md:30]: there are just so many `to`s in a row here. It's fine but maybe consider a light reword?

**Review by cgreene — APPROVED** [review]
One brief note 🚀

File: content/08.transmission.md (hunk 1 above)
Comment (rando2) [reply content/08.transmission.md:30]: Hahaha in retrospect, also some casual tautology?
```suggestion
These mechanisms may differ in their relevance to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.
```

Comment (cgreene) [reply content/08.transmission.md]: 👍

**Pull request merged**